specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-01-10/service.json

```diff
@@ -9093,16 +9261,43 @@
           }
         },
         "hasPhysicalDisk": {
-          "description": "A value indicating whether the VM has a physical disk attached.",
-          "type": "boolean"
+          "description": "A value indicating whether the VM has a physical disk attached. String value of {SrsDataContract.PresenceStatus} enum.",
```

Comment:
Is ARM team aware of these? Changing these property types are a big breaking change not allowed for a stable API version.

Reply:
The swagger changes with bool data type were taken from our development branch and made public. Post that, the code with enum was checked-in in the service, but not in swagger. Due to this, at present our service REST API layer (which has enum type) is not in sync with swagger in this regard and this change is an attempt to fix that bug as well.
Note: No customer is using the released SDK with bool.